### DISCUSSION TITLE
faq

### DISCUSSION THREAD

**Discussion by ghost** [backend-br/desafios#61] [Q&A] [answered]

**Comment by vgeruso** [comment]
Acredito que seja aqui no discussions mesmo acho que esse canal seria para isso mesmo, abrindo tópicos e debatendo de forma amigável, mas seria bom tirar a duvida com o @gustavofreze  ele que pode dar mais instruções. Mas eu estou aberto a sugestões e para ajudar também =D

**Comment by gustavofreze** [comment, marked as answer]
@lucasnuic você pode usar o [discussions](https://github.com/orgs/backend-br/discussions) do fórum. Aqui seria para discussões relacionadas aos desafios.
